static/style.css

```diff
@@ -31,8 +32,9 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 p {
+  font-family: georgia;
```

Comment:
This won't work will it with a lowecase g? Can you copy the other font-family code that uses Georgia and serif's etc?